packages/nuxt/src/imports/module.ts

```diff
@@ -228,19 +234,66 @@ function addDeclarationTemplates (ctx: Unimport, options: Partial<ImportsOptions
 
   addTypeTemplate({
     filename: 'imports.d.ts',
-    getContents: async ({ nuxt }) => toExports(await ctx.getImports(), nuxt.options.buildDir, true),
+    getContents: async ({ nuxt }) => {
```

Comment:
we shouldn't exclude them from this file as it is still possible to import from #imports (which is what this file represents)

Reply:
I’ve removed the exclusion. Thanks for the heads-up!